source/gamemodes/BoostMod.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include "../RocketPlugin.h"
+
+
+class BoostMod : public RocketGameMode
+{
+private:
+	float boostConversionPercentage = 100.0f;
+
+	void onTick(ServerWrapper server, void* params);
+	void OnBoostPickUp(ActorWrapper caller, void* params, std::string funcName);
+	void setMaxBoost(int newBoost);
+	std::vector<TeamWrapper> getTeams();
+
+	struct BoostModValues {
+		// A suffix to append to ui elements (ImGui uses the label to cache, so
+		// you can't have elements with the same labels.
+		std::string displaySuffix = "";
```

Comment:
You don't have to initialize to an empty string as the the default construction of `std::string` is `""`
```suggestion
		std::string displaySuffix;
```